### DISCUSSION TITLE
Giving Specific Access to different tables.

### DISCUSSION THREAD

**Discussion by jindalvivek** [nocodb/nocodb#7130] [Q&A]
What I want to do is to allow specific users to upload CSV's to specific tables only. No other tables should be visible to them. Using Postgresql as a Data Source.
I've tried both the ways:
1) Giving access to specific bases (i.e. making a lot of bases). This is not the solution since each base gets access to all the tables within a database and I do not want to create multiple databases since there would exist in future relationships between the tables.
2) Giving access to specific views. Not only is it not possible without specific email access, but also cannot allow uploading CSV.

Any help would be greatly appreciated.

**Comment by dstala** [comment]
table level access control is not available yet, but is in our roadmap.

**Reply by GeoCommander** [reply]
I'd love a table level access control. This would be great add on.

edit: right now it seems like we could create a new user with only the particular table view privileges then connect that user to the nocodb then do the work
